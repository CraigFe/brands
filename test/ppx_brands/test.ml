module T : sig
  type t0 [@@deriving branded]
  type _ t1 [@@deriving branded]
  type (_, _) t2 [@@deriving branded]
  type (_, _, _) t3 [@@deriving branded]
end = struct
  type t0 [@@deriving branded]
  type _ t1 [@@deriving branded]
  type (_, _) t2 [@@deriving branded]
  type (_, _, _) t3 [@@deriving branded]
end
