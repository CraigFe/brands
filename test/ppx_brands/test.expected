type t [@@deriving branded]

include struct
  let _ = fun (_ : t) -> ()

  include Brands.Branded.Make0 (struct
    type nonrec t = t
  end)
end [@@ocaml.doc "@inline"] [@@merlin.hide]
